docs/LANGUAGE

```diff
@@ -51,3 +52,12 @@ Anope Multi Language Support
 
     All .po and .pot files should be placed in modules/third/language. Additionally an update script is provided there
     that will create .pot files and merge any changes to it with existing .po files.
+
+4) Updating a language file
+
+    Poedit (https://poedit.net/) is a popular po file editor, and we recommend using it or another editor designed to edit
+    po files (especially on Windows).
+
+    After editing the relevant file in languages/, run `update.sh` in the languages/ folder, i.e. `cd` to it and run
```

Comment:
Ideally you should run this *before* editing translations rather than afterwards as it may add new strings.